porcupine/plugins/pastebin.py

```diff
@@ -246,7 +247,66 @@ def pasting_done_callback(
         )
 
 
+def ask_are_you_sure(filename: str | None, paste_class: type[Paste]) -> bool:
+    window = tkinter.Toplevel()
+    window.title(f"Pastebin {filename}")
+
+    content = ttk.Frame(window, name="content", padding=10)
+    content.pack(fill="both", expand=True)
+    content.columnconfigure(0, weight=1)
+
+    label1 = ttk.Label(
+        content,
+        name="label1",
+        text=f"Do you want to send the content of {filename} to {paste_class.name}?",
+        wraplength=300,
+        justify="center",
+        font="TkHeadingFont",
+    )
+    label1.pack(pady=5)
+
+    label2 = ttk.Label(
+        content,
+        name="label2",
+        text="This is a bad idea if your code is not meant to be publicly available.",
+        wraplength=300,
+        justify="center",
+        font="TkTextFont",
+    )
+    label2.pack(pady=5)
+
+    var = tkinter.BooleanVar(value=True)
+    checkbox = ttk.Checkbutton(
+        content, text="Show this dialog when I try to pastebin something", variable=var
+    )
+    checkbox.pack(pady=25)
+
+    result = False
+
+    def yes() -> None:
+        nonlocal result
+        result = True
+        window.destroy()
+
+    def no() -> None:
+        window.destroy()
+
+    button_frame = ttk.Frame(content)
+    button_frame.pack(fill="x")
+    ttk.Button(button_frame, text="Yes", command=yes).pack(side="left", expand=True)
+    ttk.Button(button_frame, text="No", command=no).pack(side="left", expand=True)
```

Comment:
```suggestion
    ttk.Button(button_frame, text="Yes", command=yes).pack(side="left", expand=True, fill="x", padx=(0, 10))
    ttk.Button(button_frame, text="No", command=no).pack(side="left", expand=True, fill="x", padx=(10, 0))
```
bigger clickable area